### PR TITLE
feat(samples): add entity type apis samples

### PR DESCRIPTION
**Pull request by suryaguthikonda**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-ai-platform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #313

### CODE REVIEW

**Review by diemtvu — APPROVED** [review]

File: samples/get-entity-type-sample.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  entityTypeId,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/get-entity-type-sample.js:31]: Can you change to 5s for Get/List methods

File: samples/list-entity-types-async-sample.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ */
+
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/list-entity-types-async-sample.js:30]: 5s

File: samples/list-entity-types-sample.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+ */
+
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/list-entity-types-sample.js:30]: 5s

File: samples/list-entity-types-stream-sample.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+ */
+
+'use strict';
+
+async function main(
+  project,
+  featurestoreId,
+  location = 'us-central1',
+  apiEndpoint = 'us-central1-aiplatform.googleapis.com',
+  timeout = 300000
```
Comment (diemtvu) [inline samples/list-entity-types-stream-sample.js:30]: 5s

File: samples/get-entity-type-sample.js (hunk 1 above)
Comment (suryaguthikonda) [reply samples/get-entity-type-sample.js]: Made the respective changes

**Review by dizcology — DISMISSED** [review]

**Review by dizcology — APPROVED** [review]

**Pull request merged**